input_cl.hpp

```diff
@@ -1421,6 +1421,7 @@ inline cl_int getInfoHelper(Func f, cl_uint name, T* param, int, typename T::cl_
 #if defined(CL_VERSION_1_1)
 #define __PARAM_NAME_INFO_1_1(F) \
     F(cl_context_info, CL_CONTEXT_NUM_DEVICES, cl_uint)\
+    \

```

Comment:
This looks like an accidental (unnecessary change)?

Reply:
Fixed